src/tools/pub/cmd/pub-outdated.md

```diff
@@ -0,0 +1,240 @@
+---
+title: pub outdated
+description: Use pub outdated to help you update your package dependencies.
+diff2html: true
+---
+
+_Outdated_ is one of the commands of the [pub tool](/tools/pub/cmd).
+
+{% prettify nocode tag=pre+code %}
+$ pub outdated [options]
+{% endprettify %}
+
+Use `pub outdated` to identify out-of-date [package dependencies][]
+and get advice on how to update them.
+[Best practices for dependency management][best practices]
+include using the most recent stable package versions,
+so you can get the latest bug fixes and improvements.
+
+{{site.alert.version-note}}
+  The `pub outdated` command was introduced in Dart 2.8.
+{{site.alert.end}}
+
+## Overview
+
+Here's how you can use `pub outdated` to help you
+update the dependencies of a package that you own
+(whether it's an app or library package):
+
+1. If your package doesn't have a `pubspec.lock` file
+   checked into source control,
+   **run `pub get`** in the top directory of the package —
+   the directory that contains your package's
+  [`pubspec.yaml`](/tools/pub/pubspec) file.
+1. **Run `pub outdated`**
+   to identify which package dependencies are out-of-date.
+   Note the affected packages,
+   so that later you can test the behavior of code that uses them.
+1. Follow the recommendations of `pub outdated` for updating the packages.
+   Some updates might require only running `pub upgrade`.
+   Others might require updating `pubspec.yaml` before running `pub upgrade`.
+1. **Run `pub outdated`** to confirm that you've updated all the dependencies.
+1. **Test** your package to confirm that it still works as expected.
+
+You might still have [transitive dependencies][] that are out-of-date.
+If you want to determine the cause,
+try running [`pub deps`][] and searching the output for
+the name of the out-of-date package.
+
+
+## Example
+
+Here's an example of running `pub outdated` on
+an example that depends on old versions of two packages:
+`args` and `pedantic`.
+As the following screenshot shows,
+`pub outdated` colorizes the output by default
+when you run it on the command line.
+
+![screenshot of pub outdated output; visible as text later in "Output columns" section](images/pub-outdated.png)
```

Comment:
I find the colors hard to see in this screenshot with the green background. I think it's easier with a black background.

Reply:
I tried a black background but couldn't see it due to https://github.com/dart-lang/pub/issues/2473.

If you want to take screenshots, I could use yours...

Reply:
Oh, that's sad. I'm using this Terminal, that has the colors as expected: https://www.iterm2.com/. Do you want to try that one, or should I grab some for you?

Reply:
Screenshot here with black if you need one:

https://user-images.githubusercontent.com/13644170/81102585-24ed4680-8f10-11ea-8d9f-f08466892c87.png

<img width="709" alt="Screenshot 2020-05-05 at 20 36 39" src="https://user-images.githubusercontent.com/13644170/81102585-24ed4680-8f10-11ea-8d9f-f08466892c87.png">